main.go

```diff
@@ -263,6 +263,11 @@ func processGitHubWebhook(ctx *gin.Context, payload []byte, githubClient *github
 			log.Errorf("Failed to stop a stale build after the PR: %v was merged or closed. Error: %v", pr, err)
 		}
 
+		// Delete merged pr branches in GitLab
+		if err = deleteStaleGitlabPRBranch(log, pr, conf); err != nil {
+			log.Errorf("Failed to delete the stale PR branch after the PR: %v was merged or closed. Error: %v", pr, err)
+		}
+
```

Comment:
Similar to my previous comment, this should be done before line 248, as from there on the processing only continues for `mendersoftware` members (iow mender-qa pipeline)